jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -331,15 +389,64 @@ static String getDefaultName(String method, MapArray<String, Object> args) {
     static MapArray<String, Object> classFields(Object obj) {
         return obj != null ? getAllFields(obj) : new MapArray<>();
     }
+    static MapArray<String, Object> classMethods(Object obj) {
+        return obj != null ? getMethods(obj) : new MapArray<>();
+    }
+    private static MapArray<String, Object> getMethods(Object obj) {
+        return new MapArray<>(obj.getClass().getMethods(),
+                method -> method.getName() + "()", v -> func(obj, v), true);
+    }
+    private static JFunc<String> func(Object obj, Method m) {
+        return () -> m.invoke(obj).toString();
+    }
     static String getElementName(JoinPoint jp) {
         try {
             Object obj = jp.getThis();
             if (obj == null) return jp.getSignature().getDeclaringType().getSimpleName();
             return isInterface(getJpClass(jp), INamed.class)
                 ? ((INamed) obj).getName()
                 : obj.toString();
-        } catch (Exception ex) {
-            throw exception(ex, "Can't get element name");
+        } catch (Throwable ex) {
+            return "Can't get element name";
+        }
+    }
+    static String getElementContext(JoinPoint jp) {
+        try {
+            Object obj = jp.getThis();
+            if (obj == null) return jp.getSignature().getDeclaringType().getSimpleName();
+            if (isInterface(getJpClass(jp), IBaseElement.class))
+                return ((IBaseElement) obj).base().printFullLocator();
+            return isInterface(getJpClass(jp), INamed.class)
+                    ? ((INamed) obj).getName()
+                    : obj.toString();
+        } catch (Throwable ex) {
+            return "Can't get context locator";
+        }
+    }
+    static String getFullInfo(JoinPoint jp) {
+        try {
+            Object obj = jp.getThis();
+            if (obj == null) return jp.getSignature().getDeclaringType().getSimpleName();
+            if (isInterface(getJpClass(jp), IBaseElement.class))
+                return ((IBaseElement) obj).base().toString();
+            return isInterface(getJpClass(jp), INamed.class)
+                    ? ((INamed) obj).getName()
+                    : obj.toString();
+        } catch (Throwable ex) {
+            return "Can't get context locator";
+        }
+    }
+    static String getElementLocator(JoinPoint jp) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)